webapp/src/components/input.jsx

```diff
@@ -16,8 +16,8 @@ export default class Input extends PureComponent {
             PropTypes.string,
             PropTypes.number,
         ]),
-        addValidate: PropTypes.func.isRequired,
-        removeValidate: PropTypes.func.isRequired,
+        addValidate: PropTypes.func,
+        removeValidate: PropTypes.func,
```

Comment:
I wonder why this change.

Reply:
@larkox Because there are some cases where these props aren't being passed to this component, e.g. in `AttachIssueModal`. This change is actually out of this PR's scope but I was getting warnings on the console so I decided to fix it.